src/components/Main/NavigationBar/NavigationContent/ChannelsTab.vue

```diff
@@ -66,9 +69,30 @@ const staredChannels = computed(() => {
     [...staredChannelSet.value],
     channelsMap.value
   )
-  return filterTrees(trees, channel => !channel.archived)
+  const sortedTrees = sortChannelTree(trees)
+  return filterTrees(sortedTrees, channel => !channel.archived)
 })
 
+const sortChannelTree = (tree: ChannelTreeNode[]): ChannelTreeNode[] => {
+  const mapped = tree.map((node, index) => {
+    return { index, pathString: channelIdToPathString(node.id).toUpperCase() }
+  })
```

Comment:
好みかもしれないけどちょっと簡潔にこう書けると思います(composablesの方も同様)
```suggestion
  const mapped = tree.map((node, index) => ({
    index,
    pathString: channelIdToPathString(node.id).toUpperCase()
  }))
```

Reply:
確かにこっちのほうがスッキリしますね